src/ui/components/UserReview/styles.scss

```diff
@@ -25,10 +30,11 @@
 
 .UserReview-body {
   font-size: $font-size-default;
+  margin: 0 !important;
```

Comment:
We don't tend to use `!important`. Why is it needed here?

Reply:
![mozila1](https://user-images.githubusercontent.com/50075905/73866773-6d428d00-486b-11ea-96cb-edd3d8fd6877.png)
There is an extra space between ratings and delete rating because in `.Addon .Card` selector `margin-bottom: 24px;`. is applied.